index.bs

```diff
@@ -1620,18 +1640,27 @@ To <dfn for="WebTransportSendStream">write</dfn> |chunk| to a {{WebTransportSend
 
      Note: The definition of fairness here is [=implementation-defined=].
 
-  1. If the previous step failed, abort the remaining steps.
+     Stop sending |bytes| if |atomic| is true and the current [=flow control=] window
```

Comment:
The fact that the instruction to stop comes after the instruction to send seems to imply that the chunk may be partially sent before stopping. Based on the name `atomic` I guess that is not the intention.

Reply:
As written, nothing protects the server from receiving a partial message. How should it work differently, say with a 100 MB chunk?

Reply:
I think that Adam's concern here is similar to mine.  You have a bunch of text about interleaving and whatnot, which implies that maybe you send atomic data in part.  The atomic bit really needs to come before all of that.

I would move the atomic piece up to line 1619, ahead of all of the other stuff.

Reply:
@martinthomson Thanks for clarifying. So the intent truly is to guarantee nothing is sent unless all can be sent? This would seem to limit atomicWrite chunk sizes to < the current flow control window, but I'm OK with that.